react_native/src/pages/my-page/ProfileEditPage.tsx

```diff
@@ -0,0 +1,95 @@
+import { semanticColor } from '@/styles/semantic-color';
+import DropDown from '@/components/common/dropdown/DropDown';
+import SectionSeparator from '@/components/common/SectionSeparator';
+import SettingItemRow from '@/components/my-page/SettingItemRow';
+import Chip from '@/components/common/Chip';
+import MainButton from '@/components/common/button/MainButton';
+import { useNavigation } from '@react-navigation/native';
+import TextField from '@/components/common/textField/TextField';
+import { useState } from 'react';
+
+// 프로필 사진 변경 버튼에 svg파일 추가 기능 필요
+// TextField 아래에 뜨는 텍스트 기능 확인 필요
+// DropDown 에 isPlused 안먹힘 + 테두리, placeholder 색상 수정 필요
+function ProfileEditPage() {
+  const navigation = useNavigation();
+  const [nickname, setNickname] = useState<string>('');
+
+  return (
+    <View style={styles.profileEditPageContainer}>
+      <ScrollView>
+        <ButtonTitleHeader
+          title="내 정보"
+          leftChilds={{
+            icon: <IconChevronLeft width={28} height={28} stroke={semanticColor.icon.primary} strokeWidth={2} />,
+            onPress: () => navigation.goBack(),
+          }}
+        />
+        <View style={styles.profileImageSection}>
+          <View style={styles.profileImage} />
+          <View style={styles.editButtonWrapper}>
+            <VariantButton children="프로필 사진 변경" theme="sub" onPress={() => {}} />
+          </View>
+        </View>
+        <View style={styles.textFieldSection}>
+          <TextField
+            inputText={nickname}
+            label="닉네임"
+            placeholder="닉네임"
+            setInputText={nickname => setNickname(nickname)}
+          />
+          <DropDown title="지역" placeholder="선택" isPlused />
+        </View>
+        <SectionSeparator type="line-with-padding" />
+        <SettingItemRow
+          itemName="본인인증"
+          subItem={<Chip text="완료" variant="condition" />}
+          showNextButton
+          onPress={() => {}}
+        />
+      </ScrollView>
+      <View style={styles.saveButtonWrapper}>
+        <MainButton children="저장하기" onPress={() => {}} disabled />
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  profileEditPageContainer: {
+    flex: 1,
+    backgroundColor: semanticColor.surface.white,
+  },
+  profileImageSection: {
+    paddingTop: semanticNumber.spacing[36],
+    paddingBottom: semanticNumber.spacing[12],
+    justifyContent: 'center',
+    alignItems: 'center',
+    gap: semanticNumber.spacing[8],
+  },
+  profileImage: {
+    width: 96,
+    height: 96,
+    backgroundColor: semanticColor.surface.gray,
+    borderRadius: 9999,
+  },
+  editButtonWrapper: {
+    paddingVertical: semanticNumber.spacing[8],
+  },
+  textFieldSection: {
+    padding: semanticNumber.spacing[16],
+    gap: semanticNumber.spacing[20],
+  },
+  saveButtonWrapper: {
+    paddingHorizontal: semanticNumber.spacing[16],
+    paddingTop: semanticNumber.spacing[10],
+    paddingBottom: semanticNumber.spacing[32],
```

Comment:
```suggestion
    paddingVertical: semanticNumber.spacing[10],
```
SafeArea에 대한 QA 반영이 덜 되었습니다!